<launch>
  <node ns="ipem_module" name="ipem_reactor" pkg="ipem_module" type="IPEM_pipe_reactor_2.py" output="screen">
    <remap from="source_stream" to="/binaural_audio/source_stream"/>
  </node>
</launch>
